src/scripts/quests/QuestHelper.ts

```diff
@@ -86,16 +86,29 @@ class QuestHelper {
                 amount = SeededRand.intBetween(100, 500);
                 return new UseOakItemQuest(oakItem, amount);
             case 'HarvestBerriesQuest':
+                const berryRegionBound = Farming.genBounds[Math.min(player.highestRegion() + 1, 5)];
```

Comment:
Remove `+1` addition to `highestRegion`.
Better not to use magic numbers where possible.
```suggestion
                const berryRegionBound = Farming.genBounds[Math.min(player.highestRegion(), GameConstants.Region.unova)];
```